gems/activerecord/6.0/activerecord.rbs

```diff
@@ -51,6 +51,7 @@ module ActiveRecord
     def self.create!: (**untyped) -> instance
     def self.validate: (*untyped, ?if: condition[instance], ?unless: condition[instance], **untyped) -> void
     def self.validates: (*untyped, ?if: condition[instance], ?unless: condition[instance], **untyped) -> void
+    def self.create_with: (nil | Hash[untyped, untyped]) -> Relation
```

Comment:
Could you move this into `ActiveRecord::Base::ClassMethods`, please?
And please copy it onto `_ActiveRecord_Relation_ClassMethods` too.

The rbs_rails gem uses these components and effects the subclasses of ActiveRecord::Base.

Reply:
I missed both of them. I did it.
Thanks for your review!